src/nvim/normal.c

```diff
@@ -4931,7 +4931,8 @@ static void nv_ident(cmdarg_T *cap)
         snprintf(buf, buf_size, ".,.+%" PRId64, (int64_t)(cap->count0 - 1));
       }
 
-      STRCAT(buf, "! ");
+      do_cmdline_cmd("tabnew");
+      STRCAT(buf, "term ");
```

Comment:
avoid abbreviated ex command names in scripts/core. Does the calcuatlion of `buf_size` need to change?

Reply:
>  Does the calcuatlion of buf_size need to change?

It's currently `n*2 + 30 + STRLEN(&keywordprg)` where `n` is the length of the identifier being queried. I don't know how or why this number was decided on for the buffer size but we are still within these bounds.